string camera_id
uint64 image_counter
uint8[] data
uint16 height
uint16 width
